packages/api/src/auth/githubClient/github.client.ts

```diff
@@ -0,0 +1,43 @@
+import { Injectable } from '@nestjs/common';
+import { PassportStrategy } from '@nestjs/passport';
+import { Strategy } from 'passport-github2';
+import { VerifiedCallback } from 'passport-jwt';
+
+import { UsersService } from '../../users/users.service';
+import { JwtStrategy } from '../jwtStrategy/jwt.strategy';
+import { GithubResponse, GithubUserData } from './github.model';
+
+@Injectable()
+export class GithubClient extends PassportStrategy(Strategy) {
```

Comment:
As this class extends `PassportStrategy`, I think `GithubStrategy` would be a more suitable name. This way we will avoid the strange path where you have `github.client` inside `githubClient`.
When it comes to naming I would rename `githubClient` directory to be simply `github`, but that's only a suggestion.